static/assets/geojson/hb-layers/bicyclerepairstation.geojson

```diff
@@ -0,0 +1,216 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "name_en": "Bicycle repair station",
+        "name_de": "Fahrradreparaturstation",
+        "name": "Fahrradreparaturstation",
```

Comment:
The latest comments on the layer generations seem not to be applied yet: No comma after last property, osm name should be part of feature name (applies to all layer files)